compose/snippets/src/main/java/com/example/compose/snippets/glance/GlanceSnippets.kt

```diff
@@ -0,0 +1,994 @@
+import androidx.glance.appwidget.GlanceAppWidgetReceiver
+import androidx.glance.appwidget.RadioButton
+import androidx.glance.appwidget.RadioButtonDefaults
+import androidx.glance.appwidget.SizeMode
+import androidx.glance.appwidget.Switch
+import androidx.glance.appwidget.SwitchDefaults
+import androidx.glance.appwidget.action.ActionCallback
+import androidx.glance.appwidget.action.actionRunCallback
+import androidx.glance.appwidget.action.actionSendBroadcast
+import androidx.glance.appwidget.action.actionStartService
+import androidx.glance.appwidget.provideContent
+import androidx.glance.appwidget.updateAll
+import androidx.glance.appwidget.updateIf
+import androidx.glance.background
+import androidx.glance.color.ColorProvider
+import androidx.glance.layout.Alignment
+import androidx.glance.layout.Column
+import androidx.glance.layout.Row
+import androidx.glance.layout.RowScope
+import androidx.glance.layout.fillMaxSize
+import androidx.glance.layout.fillMaxWidth
+import androidx.glance.layout.padding
+import androidx.glance.material3.ColorProviders
+import androidx.glance.text.Text
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import com.example.compose.snippets.R
+import com.example.compose.snippets.layouts.MainActivity
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+lateinit var LightColors: ColorScheme
+lateinit var DarkColors: ColorScheme
+
+private object GlanceCreateAppWidgetSnippet01 {
+    // [START android_compose_glance_receiver01]
+    class MyAppWidgetReceiver : GlanceAppWidgetReceiver() {
+        override val glanceAppWidget: GlanceAppWidget = TODO("Create GlanceAppWidget")
+    }
+    // [END android_compose_glance_receiver01]
+}
+
+private object GlanceCreateAppWidgetSnippet02 {
+    // [START android_compose_glance_receiver02]
+    class MyAppWidgetReceiver : GlanceAppWidgetReceiver() {
+        
+        // Let MyAppWidgetReceiver know which GlanceAppWidget to use
+        override val glanceAppWidget: GlanceAppWidget = MyAppWidget()
+    }
+    // [END android_compose_glance_receiver02]
+
+    // [START android_compose_glance_widget]
+    class MyAppWidget : GlanceAppWidget() {
+
+        override suspend fun provideGlance(context: Context, id: GlanceId) {
+
+            // In this method, load data needed to render the AppWidget.
+            // Use `withContext` to switch to another thread for long running
+            // operations.
+
+            provideContent {
+                // create your AppWidget here
+                Text("Hello World")
+            }
+        }
+    }
+    // [END android_compose_glance_widget]
+}
+
+private object CreateUI {
+    // [START android_compose_glance_createui]
+    /* Import Glance Composables
+     In the event there is a name clash with the Compose classes of the same name,
+     you may rename the imports per https://kotlinlang.org/docs/packages.html#imports
+     using the as keyword.
```

Comment:
Nit: maybe surround ticks around `as` to make it clear that it's the `as` reserved keyword?